src/main/java/org/commcare/formplayer/beans/AuthenticatedRequestBean.java

```diff
@@ -100,4 +101,14 @@ public String getRestoreAsCaseId() {
     public void setRestoreAsCaseId(String restoreAsCaseId) {
         this.restoreAsCaseId = restoreAsCaseId;
     }
+
+    @JsonGetter(value = "windowWidth")
+    public String getWindowWidth() {
```

Comment:
Not having much expertise here. But `AuthenticatedRequestBean` to me seems like the wrong place to put this. What is the reasoning for this location?

Reply:
Also not an expert here. Originally this was in SessionNavigationBean, but I shifted it up the inheritance tree so it would be available [here](https://github.com/dimagi/formplayer/blob/26fcd44ff99a07225b0e5153b54295e2668b4c3b/src/main/java/org/commcare/formplayer/services/MenuSessionFactory.java#L241) where it creates a new `MenuSession`.

Reply:
Probably better to add it in `InstallRequestBean` in that case.

Reply:
makes sense, fine to be in `AuthenticatedRequestBean` in that case.